samples/meetings-audio-state/python/app.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, render_template, send_from_directory, jsonify, request
```

Comment:
Please update the copyright line and ensure it is added in all missing places, refer existing samples